ddterm/shell/extension.js

```diff
@@ -245,8 +240,28 @@ function enable() {
         watch_window(actor.meta_window);
     });
 
-    desktop_entry = new DesktopEntry();
+    desktop_entry = new InstallableResource(
+        Me.dir.get_child('ddterm').get_child('com.github.amezin.ddterm.desktop'),
+        Gio.File.new_for_path(GLib.build_filenamev(
+            [
+                GLib.get_user_data_dir(),
+                'applications',
+                `${APP_ID}.desktop`,
+            ]))
+    );
     desktop_entry.install();
+
+    dbus_service = new InstallableResource(
+        Me.dir.get_child('ddterm').get_child('org.gnome.Shell.Extensions.ddterm.service'),
+        Gio.File.new_for_path(GLib.build_filenamev(
+            [
+                GLib.get_user_runtime_dir(),
+                'dbus-1',
+                'services',
+                'org.gnome.Shell.Extensions.ddterm.service',
+            ]))
+    );
+    dbus_service.install();
```

Comment:
d-bus daemon docs for `$XDG_RUNTIME_DIR/dbus-1/services`:

> Also unlike the other standard session service directories, this directory is never monitored with inotify(7) or similar APIs. Programs that create service files in this directory while a dbus-daemon is running are expected to call the dbus-daemon's ReloadConfig() method after they have made changes.

https://dbus.freedesktop.org/doc/dbus-daemon.1.html

Reply:
Thanks, I did not know that! Done in 2f0acb9.